Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -0,0 +1,109 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  SUPPORTED_REPOS = [
+    %w[brew core cask],
+    OFFICIAL_CMD_TAPS.keys.map { |t| t.delete_prefix("homebrew/") },
+    OFFICIAL_CASK_TAPS.reject { |t| t == "cask" },
```

Comment:
I don't think this needs to block this PR, but isn't `OFFICIAL_CASK_TAPS` still missing `cask-drivers` and `cask-fonts`?

Reply:
Yeah, I don't know why those don't exist. Casks aren't my area of expertise. I'll do a separate PR to add them and see what folks say.